run.cmd

```diff
@@ -0,0 +1,25 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+if not defined VisualStudioVersion (
+  if defined VS140COMNTOOLS (
+    call "%VS140COMNTOOLS%\VsDevCmd.bat"
+    goto :Run
+  )
+  echo Error: Visual Studio 2015 required.
+  echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md for build instructions.
+  exit /b 1
+)
+
+:Run
+powershell -NoProfile -NoLogo -Command "%~dp0build.cmd -NoBuild; exit $LastExitCode;"
+set _toolRuntime=%~dp0build_tools
+set _dotnet=%_toolRuntime%\dotnetcli\dotnet.exe

```

Comment:
Again, this needs to be unified for CLI repo. I understand that it's different than other repos, but we cannot build the product with the same constraints as consumers of the product.